topics.md

```diff
@@ -5,17 +5,17 @@ Interested in speaking at an upcoming TechTalk DC event? Add your information he
 This is meant to be a guide -- don't let it overwhelm you. Give us what you've got and we'll get the rest before we need it.
 
 
- * **Session Title**:  
- * **Preferred Timeframe**:  
- * **Event Type**: [Pick one: 50-min Lecture, Lightning Talk, 2-3 Hour Workshop, All Day Workshop (plan for a Saturday)]
- * **Summary**:
- * **Presenter Name(s)**:
- * **Presenter E-mail(s)**:
- * **Presenter GitHub Account**:
- * **Presenter Blog**:
- * **Audience Skill Level**: [All Skill Levels, Beginners, Intermediate, Advanced]
- * **Requirements**: [A/V, room setup, or any other needs]
- * **Code of Conduct**: I have read, understand, and agree to follow the Tech Talk DC Code of Conduct.
+ * **Session Title**: Here's What's Trending In Software Engineering
+ * **Preferred Timeframe**: Any weeknight
+ * **Event Type**: Lightning Talk (~30 minutes with questions)
```

Comment:
Looking forward to hearing it! Also learning about Oprah.

![image](https://i.giphy.com/VhFps32TlNgsg.gif)